api/scheduler_config_handler.go

```diff
@@ -6,6 +6,7 @@ import (
 
 	"github.com/Sirupsen/logrus"
 	"github.com/topfreegames/maestro/models"
+	ghodssYaml "github.com/ghodss/yaml"
```

Comment:
This import is not necessary. You can convert to json using the encoding/json package.

Reply:
Even if the input byte stream is a YAML text?

Reply:
yes